apis/errors/error.go

```diff
@@ -31,7 +32,15 @@ type Error struct {
 // Error implements the error interface
 func (e Error) Error() string {
 	if e.err != nil {
-		return e.err.Error()
+		internalErr := e.err.Error()
+		if len(e.lsErr.Message) > 0 {
+			if !strings.HasSuffix(e.lsErr.Message, internalErr) {
+				// if the developer has not already included the original error in the message, let's add it
+				return fmt.Sprintf("%s: %s", e.lsErr.Message, internalErr)
```

Comment:
```suggestion
				return return fmt.Sprintf("Op: %s - Reason: %s - Message: %s", e.lsErr.Operation, e.lsErr.Reason, fmt.Sprintf("%s: %s", e.lsErr.Message, internalErr))
```

I'm not sure if I really completely understood the discussion but aren't we loosing the operation and the reason if there is an internal error?

Reply:
Yes. But I didn't change that, it was already implemented the way that operation and reason are only returned if there is no internal error.
In the installation resource, the reason and operation fields are read, if the error is a LsError, and put into the respective fields of the installation's status. So this is only a problem for the logs, I guess ...

Reply:
But I agree with you that it would be nice to have the information for the logs too. Only ugly thing is that it will be duplicated in the installation's status, if we dump it into the error message, but that shouldn't be much of a problem.
What do you think?

Reply:
I would prefer to start with providing more information even if there are duplicates. We could still optimise long or unneeded information later but I think missing info in a support case is more critical.